R/copy_SS_inputs.R

```diff
@@ -102,8 +102,12 @@ copy_SS_inputs <- function(dir.old = NULL,
       to = file.path(dir.new, starter[["ctlfile"]]),
       overwrite = overwrite
     )
+    # check data new file could be data.ss_new or data_echo.ss_new
+    dat_name <- 
```

Comment:
In the other function on line 1452you call it `datname` instead of `dat_name`. Also, this is really trivial but it would be good to start making helper functions, such as find_datafilename <- function(dir) {} rather than having the same code in two places.

Reply:
thanks! I made both functions use datname, and also made the code into a helper function